### ISSUE TITLE
Name is not set for folium.FeatureGroup in LayerControl

### ISSUE THREAD

**Issue by YuanxiangFranck** [python-visualization/folium#397]
Hello,

I used FeatureGroup to create muliple layer for the LayerCopntrole menu
```python
# FeatureGroup.__init__(self, name=None, overlay=True, control=True)
feature_group = folium.FeatureGroup(name='something')
# add things ...
map_osm.add_children(folium.LayerControl())
```

The solution is to set the name in : feature_group.layer_name
```python
feature_group = folium.FeatureGroup()
feature_group.layer_name='something'
# add things ...
map_osm.add_children(folium.LayerControl())
```

I think this line should be added in the constructor:
```python
self.layer_name = name if name is not None else self.get_name()
```

Tested on folium 0.2.0

Franck

**Comment by YuanxiangFranck** [comment]
pull request: #397

**Comment by eddies** [comment]
I believe this was reported/addressed already: #371 (daf7a44)

**Comment by ocefpaf** [comment]
Yep. I need to finish the bugfix release to get this out there. Thanks @YuanxiangFranck for reporting this and thanks @eddies for clearing this up!
